TestApp/ios/Podfile.lock

```diff
@@ -288,18 +290,20 @@ DEPENDENCIES:
   - "RNCAsyncStorage (from `../node_modules/@react-native-community/async-storage`)"
   - RNFS (from `../node_modules/react-native-fs`)
   - RNGestureHandler (from `../node_modules/react-native-gesture-handler`)
+  - RNScreens (from `../node_modules/react-native-screens`)
   - Yoga (from `../node_modules/react-native/ReactCommon/yoga`)
 
 SPEC REPOS:
-  https://github.com/CocoaPods/Specs.git:
+  https://cdn.cocoapods.org/:
     - AppCenter
+  https://github.com/CocoaPods/Specs.git:
```

Comment:
This url was also changed automatically. Do I need to change it back?

Reply:
This is also ok